src/tests/Microsoft.PowerFx.Core.Tests/TestRunnerTests/TestRunnerTests.cs

```diff
@@ -66,82 +66,6 @@ public void TestOverride()
             Assert.Equal("expected_result2", tests[1].Expected);
         }
 
-        [Fact]
-        public void TestList()
```

Comment:
>TestList [](http://example.com/codeflow?start=20&length=8)

We may need to keep these tests. 
Especially for SQL comparisons.

Reply:
Most test cases were kept and a few were changed to adapt to the new error format